### ISSUE TITLE
Manage Domains: Oops! This is awkward :S

### ISSUE THREAD

**Issue by Antiarchitect** [shlinkio/shlink-web-client#497]
shlinkio/shlink-web-client:3.3.0 docker image with one preconfigured server via env variables (server.json inside looks valid).
+ shlinkio/shlink:2.8.1
Going to Manage Domains in web cllient, getting this in JS console:

```
Error: The target 'domainEditcompanyname.com' could not be identified in the dom, tip: check spelling
    at j (utils.js:235)
    at M (utils.js:251)
    at t.n.updateTarget (TooltipPopoverWrapper.js:303)
    at t.n.componentDidMount (TooltipPopoverWrapper.js:97)
    at hs (react-dom.production.min.js:5445)
    at Pu (react-dom.production.min.js:6687)
    at t.unstable_runWithPriority (scheduler.production.min.js:289)
    at Vo (react-dom.production.min.js:2868)
    at Mu (react-dom.production.min.js:6470)
    at mu (react-dom.production.min.js:6109)
```

P.S. I have 3-rd level domain like company.name.com

**Comment by mannp** [comment]
Thanks for reporting .. I am getting the same error.

**Comment by acelaya** [comment]
I have just released v3.3.1, which fixes this issue.

The docker image will be available after this build ends https://github.com/shlinkio/shlink-web-client/runs/3725364926?check_suite_focus=true

I will deploy the fix in app.shlink.io tomorrow.

**Comment by Antiarchitect** [comment]
@acelaya Thank you so much!
